### DISCUSSION TITLE
Is there a way to add note directly to metadata of a record?

### DISCUSSION THREAD

**Discussion by kechan** [doccano/doccano#1806] [Ideas]
I am aware this may be doable through comment? If i try to go to Comment section, the comment doesn't seem to carry around the context with it (e.g. which record this is from?). And I also don't see any UI to export the comments? 

My goal is to just make small note to remind myself to come back and investigate a particular piece of text later.

**Comment by david-engelmann** [comment]
If you utilize the doccano-client, it's possible to get inputs with their comments.

I wrote a small script that demonstrates how you could get the sample and comments for samples with comments

```
doccano_client.login(username=doccano_username,
        password=doccano_password)

examples = list(doccano_client.list_examples(project_id=1))

results = []
for example in examples:
    if example.comment_count > 0:
        comments = doccano_client.list_comments(project_id=1,
                example_id=example.id)
        results.append((example,list(comments)))
```

Hope this helps!
